depot/containerstore/containerstore.go

```diff
@@ -336,11 +336,14 @@ func (cs *containerStore) Metrics(logger lager.Logger) (map[string]executor.Cont
 			diskUsage = gardenMetric.DiskStat.TotalBytesUsed
 		}
 		containerMetrics[guid] = executor.ContainerMetrics{
-			MemoryUsageInBytes: gardenMetric.MemoryStat.TotalUsageTowardLimit,
-			DiskUsageInBytes:   diskUsage,
-			MemoryLimitInBytes: nodeInfo.MemoryLimit,
-			DiskLimitInBytes:   nodeInfo.DiskLimit,
-			TimeSpentInCPU:     time.Duration(gardenMetric.CPUStat.Usage),
+			MemoryUsageInBytes:                  gardenMetric.MemoryStat.TotalUsageTowardLimit,
+			DiskUsageInBytes:                    diskUsage,
+			MemoryLimitInBytes:                  nodeInfo.MemoryLimit,
+			DiskLimitInBytes:                    nodeInfo.DiskLimit,
+			TimeSpentInCPU:                      time.Duration(gardenMetric.CPUStat.Usage),
+			ContainerAgeInNanoseconds:           uint64(gardenMetric.Age),
+			AbsoluteCPUUsageInNanoseconds:       gardenMetric.CPUStat.Usage,
```

Comment:
Apart from the type, isn't this the same information as the `TimeSpentInCPU` value? If so, should we avoid introducing this new field and instead have the consumers of this type use the existing `TimeSpentInCPU` field?

Reply:
@emalm I'm guessing you are asking how `AbsoluteCPUUsageInNanoseconds` is different from `TimeSpentInCPU`. Is that right ?

Reply:
The reason to introduce the `AbsoluteCPUUsageInNanoseconds` attribute was all about readability purposes - we wanted to preserve the one-to-one mapping between properties of `ContainerMetrics`, the metric currently sent to aggregator and the new gauge metric. It is true that both `AbsoluteCPUUsageInNanoseconds` and `TimeSpentInCPU` have the same value (the CPU usage in ns that is taken from the `cpuacct` cgroup) though. As I don't have a strong opinion on that, I updated the PR to get rid of `AbsoluteCPUUsageInNanoseconds` and use `TimeSpentInCPU` instead